examples/resources/configuration/istio/sidecars.yaml

```diff
@@ -0,0 +1,292 @@
+              "MONGO",
+              "TCP",
+              "TLS"
+            ],
+            "required": true,
+            "placeholder": "placeholders.dropdown",
+          },
+          { "path": "targetPort", "simple": true },
+        ]
+      },
+      {
+        "widget": "FormGroup",
+        "path": "spec.ingress[].bind",
+        "placeholder": "placeholders.bind",
+        "simple": true
+      },
+      {
+        "widget": "FormGroup",
+        "path": "spec.ingress[].captureMode",
+        "enum": [
+          "DEFAULT",
+          "IPTABLES",
+          "NONE"
+        ],
+        "simple": true,
+        "placeholder": "placeholders.dropdown",
+      },
+      {
+        "widget": "FormGroup",
+        "path": "spec.ingress[].defaultEndpoint",
+        "placeholder": "placeholders.defaultEndpoint",
+        "required": true,
+        "simple": true
+      },
+      {
+        "widget": "FormGroup",
+        "path": "spec.egress[].port",
+        "simple": true,
+        "children": [
+          { 
+            "path": "number", 
+            "simple": true,
+            "required": true
+          },
+          { 
+            "path": "name", 
+            "widget": "Name", 
+            "showHelp": false, 
+            "simple": true ,
+            "required": true
+          },
+          {
+            "path": "protocol", 
+            "simple": true,
+            "enum": [
+              "HTTP",
+              "HTTPS",
+              "HTTP2",
+              "GRPC",
+              "MONGO",
+              "TCP",
+              "TLS"
+            ],
+            "required": true,
+            "placeholder": "placeholders.dropdown",
+          },
+          { "path": "targetPort", "simple": true },
+        ]
+      },
+      {
+        "widget": "FormGroup",
+        "path": "spec.egress[].bind",
+        "placeholder": "placeholders.bind",
+        "simple": true
+      },
+      {
+        "widget": "FormGroup",
+        "path": "spec.egress[].captureMode",
+        "simple": true,
+        "enum": [
+          "DEFAULT",
+          "IPTABLES",
+          "NONE"
+        ],
+        "placeholder": "placeholders.dropdown",
+      },
+      {
+        "widget": "SimpleList",
+        "path": "spec.egress[].hosts", 
+        "required": true,
+        "simple": true,
+        "placeholder": "placeholders.hosts"
+      },
+      {
+        "widget": "FormGroup",
+        "path": "spec.outboundTrafficPolicy",
+        "children": [{
+          "path": "mode", 
+          "name": "spec.outboundTrafficPolicy.mode",
+          "enum": [
+            "REGISTRY_ONLY",
+            "ALLOW_ANY"
+          ],
+          "placeholder": "placeholders.dropdown",
+        }]
+      }
+    ]
+  general: |-
+    {
+      "resource": {
+        "kind": "Sidecar",
+        "group": "networking.istio.io",
+        "version": "v1alpha3"
+      },
+      "urlPath": "ext-sidecars",
+      "category": "Istio",
+      "name": "Ext Sidecars",
```

Comment:
and just `Sidecars` for name?

Reply:
Gateways were also added as separate page (with ext prefix)